packages/ibm-products/src/components/ConditionBuilder/ConditionBuilderButton/ConditionBuilderButton.js

```diff
@@ -46,7 +54,7 @@ export const ConditionBuilderButton = ({
     <Tooltip
       label={label}
       align={tooltipAlign}
-      className={`${blockClass}__con-tooltip`}
+      className={`${wrapperClassName ?? ''}`}
```

Comment:
you should just be able to use `wrapperClassName` here without the conditional

Reply:
resolved